CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ## Unreleased
 
 ### Added
+- Added break continue support for taint mode
```

Comment:
Maybe better something like:

> Added support for `break` and `continue` in the dataflow engine

The "break continue" phrase may be a bit confusing, and this change also affects constant propagation.